python/cm/api.py

```diff
@@ -17,6 +17,7 @@
 from django.db import transaction
 from django.utils import timezone
 
+import cm.upgrade
```

Comment:
Why?

Reply:
I've tryed to fight with cyclic import in linters, but lose :(